kubectl-plugin/pipeline/gc.go

```diff
@@ -0,0 +1,118 @@
+package pipeline
+
+import (
+	"context"
+	"fmt"
+	"github.com/kubesphere-sigs/ks/kubectl-plugin/types"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/dynamic"
+	"time"
+)
+
+func newGCCmd(client dynamic.Interface) (cmd *cobra.Command) {
+	opt := &gcOption{
+		client: client,
+		pipelineCreateOption: pipelineCreateOption{
+			Client: client,
+		},
+	}
+	cmd = &cobra.Command{
+		Use:     "gc",
+		Short:   "Garbage collector for PipelineRuns",
+		Long:    "Clean all those old PipelineRuns by age or count",
+		PreRunE: opt.preRunE,
+		RunE:    opt.runE,
+	}
+
+	flags := cmd.Flags()
+	flags.IntVarP(&opt.maxCount, "max-count", "", 10,
```

Comment:
Add the `--verbose` parameter to provide more output information, see https://github.com/kubesphere-sigs/ks/pull/179/files#r697239943

link: http://www.linfo.org/rule_of_silence.html

Reply:
A single option `--verbose` might not be enough. I prefer to support something like `-v=7` or a logger level. IMO, this is not part of this PR. And this is not a blocker for this feature. I will improve it in the further PRs.

I've created an issue to track the rules for code review. See https://github.com/kubesphere-sigs/ks/issues/185

Thanks for all your suggestions @cndoit18 and @JohnNiang